client/post-editor/editor-action-bar/style.scss

```diff
@@ -13,6 +11,24 @@
 	}
 }
 
+.editor-action-bar__cell {
+	&.is-left,
+	&.is-right {
+		display: flex;
+		flex-shrink: 1;
+		width: 50%;
```

Comment:
Actually, in my testing, the solution for IE11 is to change this from `width: 50%;` to `flex-basis: 50%;` . If you can make that change, I think we should be good to go.